### ISSUE TITLE
DayZMod 1.8.9 Issues.

### ISSUE THREAD

**Issue by R4Z0R49** [DayZMod/DayZ#865]
Any issues with 1.8.9 please post here

**Comment by rvett** [comment]
Loot still floating. 

![floating_loot](https://user-images.githubusercontent.com/6949874/27562236-edd4e4bc-5acb-11e7-868a-abf9aeea57b0.jpg)
![floating_loot2](https://user-images.githubusercontent.com/6949874/27701447-1fac6b68-5d01-11e7-90a4-70674c80dfe7.jpg)
![floating_loot3](https://user-images.githubusercontent.com/6949874/27988658-6395ba0a-6427-11e7-81b0-b4c52e92784d.jpg)
![floating_loot4](https://user-images.githubusercontent.com/6949874/28138475-31afb842-6751-11e7-9855-f60444e22800.jpg)
![floating_loot5](https://user-images.githubusercontent.com/6949874/28138476-31b23e0a-6751-11e7-84da-e34319a5bfde.jpg)
![floating_loot6](https://user-images.githubusercontent.com/6949874/28239501-e6f243be-696d-11e7-9844-9613b18787ff.jpg)

**Comment by romanongit** [comment]
Some more floating loot:

![floating_loot2](https://user-images.githubusercontent.com/6949874/27701447-1fac6b68-5d01-11e7-90a4-70674c80dfe7.jpg)

**Comment by unhappytroll** [comment]
2 vehicles can be spawned in one place (with same world coordinates), make them spawn one inside the other. Video below shows UAZ and BAF offroad spawned in one place, near TV tower.
https://youtu.be/rLFWEQbc95A
upd. also tents can fully lose  stashed stuff sometimes, although I can't tell the conditions on it right now. Not when overloaded, just like you came to it and everything is gone. On private server with no other ppl  online except me.

**Comment by toejam119** [comment]
Before 1.8.9 version, you could do a running jump and the player would only lose some of stamina. Now if you do a running jump you'll lose 100% of the stamina and your player is 100% out of breath, even if you had 100% of stamina right before the running jump.

**Comment by Landbergaren** [comment]
you lost all your stamina before as well..

**Comment by toejam119** [comment]
I think I'm confusing the DayZ Mod running jump with the Overwatch (the one that worked with Epoch 1.0.5.1) running fence jump, where the character holds his hand on fence while doing a sideways jump over a fence, there you only lost a little bit of stamina. You had to have main weapon on while doing this. For some reason that jump is gone from the Overwatch that works with 1.0.6 - 1.0.6.1 Epoch.

I watched all my old Overpoch 1.0.5.1 videos and that was the case. Unfortunately I don't have older videos of plain Epoch 1.0.5.1 with pre 1.8.9 Mod and it's been so long I played it, so I'll take Landbergaren's word for it that the DayZ Mod running jump behaviour hasn't changed.
